src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigureBase {
+    private int firstSide;
+    private int secondSide;
+
+    public RightTriangle(String color, int firstSide, int secondSide) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create separate variables and pass them on for better code readability. This will also allow you to validate the inputs before creating the object.